modules/configuring-albo-on-sts-cluster-predefined-credentials.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-albo-on-sts-cluster-predefined-credentials_{context}"]
+= Configuring the AWS Load Balancer Operator on Secure Token Service cluster by using specific credentials
+
+You can specify the credential secret by using the `spec.credentials` field in the AWS Load Balancer Controller custom resource (CR). You can use the predefined  `CredentialsRequest` object of the controller to know which roles are required.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. Download the CredentialsRequest custom resource (CR) of the AWS Load Balancer Controller, and create a directory to store it by running the following command:
++
+[source,terminal]
+----
+$ curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/controller/controller-credentials-request.yaml
+----
+
+. Use the `ccoctl` tool to process the `CredentialsRequest` object of the controller:
++
+[source,terminal]
+----
+$ ccoctl aws create-iam-roles \
+        --name <name> --region=<aws_region> \
+        --credentials-requests-dir=<path-to-credrequests-dir> \
+        --identity-provider-arn <oidc-arn>
+----
+
+. Apply the secrets to your cluster:
++
+[source,terminal]
+----
+$ ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+----
+
+. Verify that the credentials secret of the controller is created:
```

Comment:
Suggestion: Instead of "Verify that the credentials secret of the controller is created:", replace with "Verify the credentials secret has been created for use by the controller:"

Reply:
Addressed